### ISSUE TITLE
Syling enhancement request

### ISSUE THREAD

**Issue by EN20M** [TomSchimansky/CustomTkinter#528]
Could you add passing of the `anchor` argument into the CTkOptionMenu() and CTkComboBox() or any text containing widget so one can chose the justification

**Comment by Akascape** [comment]
@EN20M Ig only CTkLabel has the anchor parameter. 

```python
label = customtkinter.CTkLabel(master=frame, text="LABEL1", anchor='w')
label.grid(pady=10, padx=10)
label2 = customtkinter.CTkLabel(master=frame, text="LABEL2")
label2.grid(pady=10, padx=10)
```
![Screenshot](https://user-images.githubusercontent.com/89206401/196234326-a975a614-ae78-4786-a9ec-3dd241948f7d.jpg)

**Comment by EN20M** [comment]
@Akascape I know and I am asking for this or a similar parameter getting exposed for the mentioned classes so one has the chance to decide where the text is displayed.

For the `CTkComboBox` its is as easy as passing the the `anchor` through to its internal use. But for the `CTkOptionMenue` neither `anchor` nor `justify` gets used internaly and I don't know if there is even a way to tell tkinter where to put the text for the classes used there.

**Comment by TomSchimansky** [comment]
I added `anchor` attribute for optionemnu and `justify` attribute to combobox, will be available with the next release 5.0.0.

**Comment by TomSchimansky** [comment]
Version 5.0.0 is now available and I will add an anchor option to switch, checkbox and radiobutton soon. https://github.com/TomSchimansky/CustomTkinter/issues/628
